scripts/README.md

```diff
@@ -48,7 +48,9 @@ Anytime there are _any changes_, you need to run the script that generates the u
 1. Is this the first time that articles have been written for this language? If so:
   * make sure this language appears in the [website config](https://github.com/InnerSourceCommons/innersourcecommons.net/blob/master/config.yaml) and there is an [index page](https://github.com/InnerSourceCommons/innersourcecommons.org/tree/master/content/en/learn/learning-path).
 3. Is there a whole new Learning Path section? If so:
-  * update the ["sections" config](https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/main/scripts/section_data.json) with the new section and open a pull request for the change
+  * Create a [new thumbnail](../assets/thumbnails) and put it with the [other thumbnails](https://github.com/InnerSourceCommons/innersourcecommons.org/tree/master/static/images/learn).
+  * Create an `index.md` in the section folder.****
+  * update the ["sections" config](https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/main/scripts/section_data.json) with the new section and open a pull request for the change.
```

Comment:
```suggestion
   * Create a [new thumbnail](../assets/thumbnails) and put it with the [other thumbnails](https://github.com/InnerSourceCommons/innersourcecommons.org/tree/master/static/images/learn).
   * Create an `index.md` in the section folder.
   * Update the ["sections" config](https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/main/scripts/section_data.json) with the new section and open a pull request for the change.
```

Adding more more space to make these bullets sub-lists of the numbered list above.

Also removed `****`, as I didn't know what that was for.